src/main.cpp

```diff
@@ -96,7 +96,7 @@ int entry(const char *name, int argc, char *argv[]) {
 */
 namespace version {
 int entry(const char *name, int argc, char *argv[]) {
-  std::cout << PROJECT_NAME << " version: v" << PROJECT_VER << std::endl;
+  std::cout << PROJECT_NAME << " version: " << PROJECT_VER << std::endl;
```

Comment:
Unless you're strongly against this, could you leave the "v" prefix as it matches the tags on releases?

Reply:
Done